packages/app/src/app/hooks/useDismissible.ts

```diff
@@ -4,24 +4,29 @@ import { useEffects } from 'app/overmind';
 /**
  * Localstorage keys used for dismissible modals and banners.
  */
-type DismissibleKeys =
+type DismissibleKeys<T extends string> =
+  | T
```

Comment:
Do you think it still makes sense to explicitly type these keys instead of just using a string if we accept `T`?

Reply:
@tristandubbeld probably not...

Reply:
removed here: [259d243](https://github.com/codesandbox/codesandbox-client/pull/7435/commits/259d2435de752bd760eb5a56b6539f0e101f6cb2)